README.md

```diff
@@ -83,48 +83,51 @@ If you came here to make changes to the generated TypeScript libraries (e.g. `@g
 you are in the right place! Chances are high you don't need to edit any code, just the
 [Nunjucks](https://mozilla.github.io/nunjucks/) templates located in the `templates` folder.
 
-After you edited the files, make the generator available globally:
+This project is written in TypeScript, but we use [Bazel](https://bazel.build/) for managing dependencies
+(for unrelated reasons), which makes the build process a little bit unusual.
+
+Good news is that you don't really need to learn Bazel to make a quick fix. All bazel commands are
+hidden under the hood, so just use regular `npm` commands and just don't be surprised to see a lot of extra
+output.
+
+To compile the code:
 
 ```sh
 # in gapic-generator-typescript folder
-$ npm install  # install dependencies
-$ npm install -g .
-# make sure gapic-generator-typescript launch script in PATH
+$ npm install      # install dependencies
+$ npm run compile  # build project with Bazel
 ```
```

Comment:
I encounter $USE_BAZEL_VERSION issue. I suggest that we add line to export $USE_BAZEL_VERSION
```sh
export USE_BAZEL_VERSION=`which bazel`
```

Reply:
It's optional. It should totally work without this variable, so not mentioning it in `README`. We'll adapt to this and then can understand what's required and what's not :)

Reply:
Actually, `npm run compile` work without set $USE_BAZEL_VERSION on my end.